proposals/lip-0059.md

```diff
@@ -61,33 +61,33 @@ Here
 ```python
 lastHeightOfRound(h):
     r = roundNumber(h)
-    return r * ROUND_LENGTH + genesisHeight
+    return r * ROUND_LENGTH + genesisDataStore.height
 ```
 
-Here `roundNumber` is a function defined in the [LIP "Define state and state transitions of DPoS module"][research:dpos-module#roundnumber] and `genesisHeight` denotes the height of the genesis block as stored in the [DPoS module][research:dpos-module].
+Here `roundNumber` is a function defined in [LIP 0057][lip-0057#roundnumber] and `genesisDataStore.height` denotes the height of the genesis block as stored in the [DPoS module][lip-0057].
```

Comment:
> `roundNumber` is a function defined in [LIP 0057]

this is maybe a bit misleading because this function is not public. So maybe we should define this internal function here as well or just replace `roundNumber(h)` by the formula.

Reply:
This LIP defines some additional functionality of the DPoS module and would therefore be implemented as part of the DPoS module. Therefore, I assume here that I can use internal functions of the DPoS module.